SystemInformer/proctree.c

```diff
@@ -2932,27 +2921,8 @@ BOOLEAN NTAPI PhpProcessTreeNewCallback(
 #endif
                 break;
             case PHPRTLC_ELEVATION:
-                {
-                    PWSTR type;
-
-                    switch (processItem->ElevationType)
-                    {
-                    case TokenElevationTypeDefault:
-                        type = L"N/A";
-                        break;
-                    case TokenElevationTypeLimited:
-                        type = L"Limited";
-                        break;
-                    case TokenElevationTypeFull:
-                        type = L"Full";
-                        break;
-                    default:
-                        type = L"N/A";
-                        break;
-                    }
-
-                    PhInitializeStringRefLongHint(&getCellText->Text, type);
-                }
+                getCellText->Text = PhGetStringRef(PhGetElevationTypeString(processItem->IsElevated ? TRUE : FALSE, processItem->ElevationType));
+                PhDereferenceObject(&getCellText->Text);
```

Comment:
`PH_STRINGREF` is not an object, you can't reference it as such. I recognize the naming might be counter-intuitive but a `PH_STRINGREF` can't assume a strong reference to the string, it's a "view" into a string. Take a look at the example above on how `PHPRTLC_RELATIVESTARTTIME` manages the string.

Reply:
hope is ok now ;)